canonical-web-and-design/ubuntu.com
# Remove /openstack/storage page

## Done

- Remove /openstack/storage page
- Redirect to /ceph
- Archived the copy doc

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/storage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are redirected to /ceph
- That it isn't in the /openstack navigation or the footer
